src/test/java/com/ibm/cloudant/kafka/connect/CloudantSourceTaskTest.java

```diff
@@ -35,138 +21,132 @@
-		
-	}
-	
-	public void testStartWithIncrementalUpdates() throws InterruptedException {
-		
-		expectOffsetLookupReturnNone();
-		
-		PowerMock.replayAll();
-
-		// Test again with a last sequence number 0 and a batch size > number of documents
-		sourceProperties.put(InterfaceConst.LAST_CHANGE_SEQ, "0");
-		sourceProperties.put(InterfaceConst.BATCH_SIZE, "1000");
-
-     	task.start(sourceProperties);
-		List<SourceRecord> records2 = task.poll();
-		
-		// We have 999 docs in the database at this point
-		assertEquals(999, records2.size());
-		
-		// Load 20 new documents
-		JSONArray data2 = new JSONArray();
-		int new_changes = 20;
-		for (int i = 0; i < new_changes; i++) {
-			data2.put(data.get(i));
-		}
-		
-		JavaCloudantUtil.batchWrite(testProperties.getProperty(InterfaceConst.URL), 
-				testProperties.getProperty(InterfaceConst.USER_NAME), 
-				testProperties.getProperty(InterfaceConst.PASSWORD), 
-				data2);
-	
-		// Poll again for changes and expect to get the 20 we just inserted
-		// (even though database has 999 + 20 documents now)
-		records2 = task.poll();
-		assertEquals(new_changes, records2.size());
-	}
-	
-	/**
-	 * @return the sourceProperties
-	 */
-	public Map<String, String> getSourceProperties() {
-		return sourceProperties;
-	}
-
-	/**
-	 * @param sourceProperties the sourceProperties to set
-	 */
-	public void setSourceProperties(Map<String, String> sourceProperties) {
-		this.sourceProperties = sourceProperties;
-	}
-
-	/**
-	 * @return the task
-	 */
-	public CloudantSourceTask getTask() {
-		return task;
-	}
-
-	/**
-	 * @param task the task to set
-	 */
-	public void setTask(CloudantSourceTask task) {
-		this.task = task;
-	}
-	
-	protected void tearDown() throws Exception {
-
-		// Remove the created database
-		CloudantDbUtils.dropDatabase(sourceProperties.get(InterfaceConst.URL), 
-				sourceProperties.get(InterfaceConst.USER_NAME), 
-				sourceProperties.get(InterfaceConst.PASSWORD));
-	}
+    private CloudantSourceTask task;
+    private Map<String, String> sourceProperties;
+
+    private JSONArray data = null;
+
+    protected void setUp() throws Exception {
+
+        super.setUp();
+
+        sourceProperties = ConnectorUtils.getSourceProperties();
+
+        /*
+         * 1. Create a database and load data
+         */
+        JSONTokener tokener = new JSONTokener(new FileReader("src/test/resources/data.json"));
+        data = new JSONArray(tokener);
+
+        // Load data into the source database (create if it does not exist)
+        JavaCloudantUtil.batchWrite(sourceProperties.get(InterfaceConst.URL),
+                sourceProperties.get(InterfaceConst.USER_NAME),
+                sourceProperties.get(InterfaceConst.PASSWORD),
+                data);
+
+        /*
+         * 2. Create connector
+         */
+        task = ConnectorUtils.createCloudantSourceConnector(sourceProperties);
+    }
+
+    private void expectOffsetLookupReturnNone() {
```

Comment:
Remove?

Reply:
Done in c11f114da76c6e7033d41a8b3e7b36184867050b
Along with some other commented out code in the tests.